src/Interface/hiopInterface.h

```diff
@@ -1,38 +1,44 @@
 // The C interface header used by the user. This needs a detailed user documentation.
 
+/** The hiop_int_type type defined below needs to match the C++ type hiop::int_type defined for 
+ * HiOp in the C++ header file hiop_defs.hpp
+ */
+typedef int hiop_int_type;
```

Comment:
Would it be possible to use `hiop::int_type` here?
```suggestion
typedef int hiop::int_type;
```

Reply:
I think this is specialized for C interface. Can we define it in hiop_defs.hpp?

Reply:
`hiopInterface.h` is C code, cannot use C++ code like `using` or including `hiop_defs.hpp` ... a common C include file with defines of  common types between C and C++ interfaces would be ideal, but the typedef should do the job for now.